test/cri-containerd/runpodsandbox_test.go

```diff
@@ -853,6 +853,74 @@ func Test_RunPodSandbox_Mount_SandboxDir_LCOW(t *testing.T) {
 	//TODO: Parse the output of the exec command to make sure the uvm mount was successful
 }
 
+func Test_RunPodSandbox_Mount_SandboxDir_WCOW(t *testing.T) {
```

Comment:
Can we add another test for one container with the mount and another without it and verify the one without doesn't have the mount?

Reply:
Sure

Reply:
Added!